easybuild/easyblocks/i/imkl.py

```diff
@@ -112,7 +112,9 @@ def make_module_req_guess(self):
                     'FPATH': ['mkl/include', 'mkl/include/fftw'],
                 }
                 if LooseVersion(self.version) >= LooseVersion('11.0'):
-                    if LooseVersion(self.version) >= LooseVersion('11.1'):
+                    if LooseVersion(self.version) >= LooseVersion('11.3'):
+                        retdict['MIC_LD_LIBRARY_PATH'] = ['lib/intel64_lin_mic', 'mkl/lib/mic'];
```

Comment:
Well, yeah, ok, doesn't really matter, just figured it would be better to include the actual directory rather than the symlink (for massive speedups ;-))